tools/integration_tests/run_e2e_tests.sh

```diff
@@ -42,10 +47,19 @@ fi
 if [ "$SKIP_NON_ESSENTIAL_TESTS_ON_PACKAGE" == true ]; then
   GO_TEST_SHORT_FLAG="-short"
   echo "Setting the flag to skip few un-important integration tests."
-  INTEGRATION_TEST_TIMEOUT=40m
+  INTEGRATION_TEST_TIMEOUT=50m
   echo "Changing the integration test timeout to: $INTEGRATION_TEST_TIMEOUT"
 fi
 
+if [ "$RUN_TESTS_WITH_PRESUBMIT_FLAG" == true ]; then
+  GO_TEST_PRESUBMIT_FLAG="-presubmit"
```

Comment:
Nit: it's not really a standard go test flag, so this can be renamed to something like `PRESUBMIT_RUN_FLAG` .

Reply:
done